spec/system/avo/exposed_methods_base_controller_spec.rb

```diff
@@ -1,6 +1,13 @@
 require "rails_helper"
 
 RSpec.feature "ExposedMethodsBaseController", type: :system do
+  around do |example|
+    old_value = Capybara.raise_server_errors
+    Capybara.raise_server_errors = false
```

Comment:
I'm not sure how those errors (our new custom ones) are triggered in the test env, because in the development it is not triggered when course is created, updated or deleted.

Reply:
Actually the error in development is there but suppressed on the frame.

![image](https://github.com/avo-hq/avo/assets/69730720/94df4b64-1c0f-42f4-b2ab-debc23e9c1f0)

I'm not totally convinced by `Capybara.raise_server_errors = false` strategy since it can omit other errors that we want the test to cover.

Can we remove the capybara server errors suppression and change the dummy app to avoid this breaks?

If some fields are needed on a specific spec we always can use the `with_temporary_items` helper

Reply:
Sure thing, we can do that. 

My intention though was to have an example of how this error would look like for each of the association type in the dummy app. Maybe we don't need to keep it and only trigger it in specs using the `with_temporary_items` helper.